google/cloud/security/common/email_templates/inventory_snapshot_summary.jinja

```diff
@@ -0,0 +1,46 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!doctype html>
+<html>
+
+<head>
+</head>
+
+<body>
+    Organization ID: {{ organization_id }}<br>
+    Timestamp: {{ cycle_timestamp }}<br>
```

Comment:
Just a question -- Would it make sense to include a human readable timestamp as well?